src/urlCapture.ts

```diff
@@ -401,8 +402,8 @@ export function getSuggestedDisplayName(baseUrl: string, usedNames: string[]) {
     if (!isIPAddress && !usedNamesSet.has(hostName.toLowerCase())) {
         return hostName;
     }
-    for (let i = 1; i < 10; i++) {
-        const name = `${hostName} ${i}`;
+    for (let i = 0; i < 10; i++) {
+        const name = `${hostName} ${i === 0 ? '' : i}`.trim();
```

Comment:
Maybe a tad bit more readable:
```suggestion
        const name = i === 0 ? hostName : `${hostName} ${i}`;
```

Reply:
Oops, will make this change in another PR